src/test/groovy/nebula/plugin/responsible/NebulaFacetPluginSpec.groovy

```diff
@@ -33,17 +31,19 @@ class NebulaFacetPluginSpec extends PluginProjectSpec {
 
         SourceSet integTest = project.sourceSets.find { it.name == 'integTest'}
         integTest
-        integTest.compileConfigurationName == 'integTestCompile'
-        integTest.compileClasspath.files.find { 'src/integTest'}
-        integTest.runtimeConfigurationName == 'integTestRuntime'
+        integTest.compileClasspathConfigurationName == 'integTestCompileClasspath'
+        integTest.compileClasspath.files.find {
+            it.toString().contains('classes/java/main')
```

Comment:
The previous condition was actually incorrect. `find` should have boolean predicate so `'src/integTest'` was just returning the first element on the compile classpath. Also, `compileClasspath` should not contain `integTest` sources but the output of the parent `main` source set.